st2mistral/tests/unit/test_filter_base.py

```diff
@@ -0,0 +1,108 @@
+# Licensed to the StackStorm, Inc ('StackStorm') under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest2
+import yaql
+
+ROOT_YAQL_CONTEXT = None
+
+
+def get_filters():
+    from st2mistral.filters import complex_type
+    from st2mistral.filters import data
+    from st2mistral.filters import json_escape
+    from st2mistral.filters import regex
+    from st2mistral.filters import time
+    from st2mistral.filters import use_none
+    from st2mistral.filters import version
+
+    return {
+        'to_json_string': data.to_json_string,
+        'to_yaml_string': data.to_yaml_string,
+
```

Comment:
No reason, just copied from st2. Fixed in https://github.com/StackStorm/st2mistral/commit/f49a9ff437e329934e85393d0ea4194822e1ad77